docs/plugins/outputs.asciidoc

```diff
@@ -55,6 +55,7 @@ The following output plugins are available below. For a list of Elastic supporte
 | <<plugins-outputs-stomp,stomp>> | Writes events using the STOMP protocol | https://github.com/logstash-plugins/logstash-output-stomp[logstash-output-stomp]
 | <<plugins-outputs-syslog,syslog>> | Sends events to a `syslog` server | https://github.com/logstash-plugins/logstash-output-syslog[logstash-output-syslog]
 | <<plugins-outputs-tcp,tcp>> | Writes events over a TCP socket | https://github.com/logstash-plugins/logstash-output-tcp[logstash-output-tcp]
+| <<plugins-outputs-timber,timber>> | Sends events to the Timber.io logging service | https://github.com/logstash-plugins/logstash-output-timber[logstash-output-timber]
```

Comment:
@jsvYou also need to add an include statement (further on down in the file...order is alphabetical) or this link won't resolve

Reply:
oh my ignorance..I'll build the docs locally before submitting more blind code to this PR, thanks Dede 👍